source/geh_common/release-notes.md

```diff
@@ -1,5 +1,9 @@
 # GEH Common Release Notes
 
+## Version 6.1.0
+
+- Add enforce_row_order default false to assert_dataframes_and_schemas, so it is possible to test row order
```

Comment:
Are you enforcing something, or are you verifying or asserting it?

Reply:
changed to ignore_row_order